src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -341,14 +341,15 @@ def create_ag_http_listener(cmd, resource_group_name, application_gateway_name,
         host_name=host_name,
         require_server_name_indication=True if ssl_cert and host_name else None,
         protocol='https' if ssl_cert else 'http',
-        ssl_certificate=SubResource(id=ssl_cert) if ssl_cert else None)
+        ssl_certificate=SubResource(id=ssl_cert) if ssl_cert else None,
+        firewall_policy=SubResource(id=firewall_policy) if firewall_policy else None)
```

Comment:
if cmd.supported_api_version(min_api='2019-09-01'):

Reply:
You are right. Fixing it.